makefile

```diff
@@ -22,13 +25,25 @@ PREFIX=/usr/local
 EXEC_CLI=fsqlf
 EXEC_GUI=wx_fsqlf
 CC=gcc
-CFLAGS+=-m32
+ARCH=-m32
 CXX=g++
 CXXFLAGS+= `wx-config --cxxflags`
 LDFLAGS+= `wx-config --libs`
+DEFINES=linux
```

Comment:
This line `DEFINES=linux` breaks the build on my machine. 
Command generated by make: `gcc -std=c99 -Wall -pedantic-errors -g linux -m32  -c core/cli.c  -o core/cli.o`
Error message from gcc: `gcc: error: linux: No such file or directory`
Could you elaborate why it was added?
(option-1 really not needed, some mistake; option-2 some good reason for adding it, just some glich in the process or my computer)